modules/ROOT/pages/salesforce/salesforce-connector-xml-maven.adoc

```diff
@@ -0,0 +1,112 @@
+----
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:sfdc="http://www.mulesoft.org/schema/mule/salesforce"
+      xsi:schemaLocation="
+               http://www.mulesoft.org/schema/mule/core
+               http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+               http://www.mulesoft.org/schema/mule/sfdc
+               http://www.mulesoft.org/schema/mule/sfdc/current/mule-salesforce.xsd">
+
+      <!-- Insert your configuration elements and your flow here -->
+</mule>
+----
+
+[[pomfile]]
+== POM File Configuration
+
+When creating an app manually from the XML, you need to set up
+the pom.xml for your project:
+
+[source,xml,linenums]
+----
+<repositories>
+   <repository>
+        <id>mule-ee-releases</id>
+        <name>MuleEE Releases Repository</name>
+        <url>https://repository-master.mulesoft.org/nexus/content/repositories/releases-ee/</url>
+    <repository>
+        <id>mule-ee-snapshots</id>
+        <name>MuleEE Snapshots Repository</name>
+        <url>https://repository-master.mulesoft.org/nexus/content/repositories/ci-snapshots/</url>
+    </repository>
+</repositories>
+----
+
+This snippet specifies a project dependency for a specific release version:
+
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>org.mule.connectors</groupId>
+    <artifactId>mule-sfdc-connector</artifactId>
+    <version>RELEASE</version>
+    <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+This snippet adds specifies a project dependency for the latest version:
+
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-sfdc-connector</artifactId>
+    <version>LATEST</version>
+</dependency>
+----
+
+[[apexml]]
+== Apex XML Settings
+
+The Salesforce https://www.salesforce.com/us/developer/docs/apexcode/[Apex SOAP API] exposes Apex class methods as custom SOAP Web service calls. This allows an external application to invoke an Apex Web service to perform an action in Salesforce.
+
+* To fetch all SOAP metadata, add the `fetchAllApexSoapMetadata="true"` attribute to the `<salesforce:sfdc-config` statement.
+* To fetch all REST metadata, add the `fetchAllApexRestMetadata="true"` attribute to the `<salesforce:sfdc-config` statement.
+* To add Apex class names, set them in the `<salesforce:apex-class-names >` block.
+
+For example:
+
+[source,example,linenums]
+----
+<salesforce:sfdc-config name="Salesforce_Config" doc:name="Salesforce Config" 
+    fetchAllApexSoapMetadata="true" 
+  fetchAllApexRestMetadata="true">
+  <salesforce:apex-class-names >
+       <salesforce:apex-class-name value="xyz" />
+  </salesforce:apex-class-names>
+</salesforce:sfdc-config>
+----
+
+== Keep Session Alive
+
+You can keep a session alive until it expires by setting `disableSessionInvalidation="true"` in the XML flow. You should keep the session alive when you are working with threads or concurrency.
```

Comment:
Don't use "You should" as auxiliary verbs are hard to translate and ambiguous in English anyway. How about:

Keep the session alive when you work with threads or concurrency. To keep a session alive, set ...

What does it mean to "work with concurrency"?